.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
I'm sure. See the build log of [Travis job 63.2](https://travis-ci.org/jgonggrijp/pip-review/jobs/194665677). It includes several lines of clear evidence that Python and Pip are running from a virtual environment in the (unprivileged) user home directory:

    141 $ source ~/virtualenv/python2.7/bin/activate
    145 $ pip --version
    146 pip 6.0.7 from /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages (python 2.7)

It could not be otherwise, anyway, because our `.travis.yml` does not state that it needs sudo rights. So we don't have system-wide install rights and we can only install anything using `pip` inside a virtual environment.

Moreover, on line 4 of the cram test we create a second virtual environment. Because of line 5, all tests are run from within that environment, so we never use the updated version of `pip` that you enforced in the `.travis`. Please remove the line.